CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [2.39.2] - 2022-02-04
```

Comment:
this can be 2.40, looks more than a patch.

Reply:
Sure, I'll fix this